renderer.js

```diff
@@ -985,14 +1011,92 @@ function playPressed(){
 
         playModalQuitDLCancel = dataObj.reqObj;
         
+        
     });
 
 }
 
-playButtonText.addEventListener("click", function(event){
+// Checks the graphics card
+async function checkIfCompatible() {
+    try {
+        const data = await si.graphics();
+
+        // Get all of the graphics cards info
+        for (var i = 0; i < data.controllers.length; i++){
+            graphicControllers = data.controllers[i];
+            cardsVendor = String(graphicControllers.vendor.toLowerCase());
+            cardsModel = String(graphicControllers.model + ", ");
+            console.log(graphicControllers.model);
+        }
```

Comment:
I tested it on an amd card laptop, and the gpu vendor info actually does says Advanced Micro Devices, Inc. But, I will add it in just in case though...